front/SettingsForm.tsx

```diff
@@ -90,6 +92,7 @@ const SettingsForm = () => {
               className="form-control form-control-lg"
               type="email"
               placeholder="Email"
+              data-cy="settings-email"
```

Comment:
The data-cy attribute value for the email input field should be 'settings-email-input' for better clarity.